### PR TITLE
ConfigureSecureBootPage: add missing tests & fix findings

### PR DESCRIPTION
**Pull request by jpnurmi**

### CODE REVIEW

File: packages/ubuntu_desktop_installer/test/configure_secure_boot_page_test.dart [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    final fields = find.widgetWithText(ValidatedFormField, 'key');
+    expect(fields, findsNWidgets(2));
+    expect(tester.widget<ValidatedFormField>(fields.first).enabled, isTrue);
+    expect(tester.widget<ValidatedFormField>(fields.last).enabled, isTrue);
+
+    final radios = find.byTypeOf<RadioButton<SecureBootMode>>();
+    expect(radios, findsNWidgets(2));
+    expect(tester.widget<RadioButton<SecureBootMode>>(radios.first).groupValue,
+        equals(SecureBootMode.turnOff));
+    expect(tester.widget<RadioButton<SecureBootMode>>(radios.first).groupValue,
```
Comment (oSoMoN) [inline packages/ubuntu_desktop_installer/test/configure_secure_boot_page_test.dart:74]: Did you mean to test `radios.last` here?

File: packages/ubuntu_desktop_installer/test/configure_secure_boot_page_test.dart [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+  testWidgets('don\'t install', (tester) async {
+    final model = buildModel(secureBootMode: SecureBootMode.dontInstall);
+    await tester.pumpWidget(buildApp(tester, model));
+
+    final radios = find.byTypeOf<RadioButton<SecureBootMode>>();
+    expect(radios, findsNWidgets(2));
+    expect(tester.widget<RadioButton<SecureBootMode>>(radios.first).groupValue,
+        equals(SecureBootMode.dontInstall));
+    expect(tester.widget<RadioButton<SecureBootMode>>(radios.first).groupValue,
```
Comment (oSoMoN) [inline packages/ubuntu_desktop_installer/test/configure_secure_boot_page_test.dart:111]: ditto

**Review by oSoMoN — COMMENTED** [review]
LGTM, just two minor comments.

File: packages/ubuntu_desktop_installer/test/configure_secure_boot_page_test.dart (hunk 1 above)
Comment (jpnurmi) [reply packages/ubuntu_desktop_installer/test/configure_secure_boot_page_test.dart]: Yes :)

**Pull request merged**